src/remote.c

```diff
@@ -1374,7 +1386,7 @@ static int update_tips_for_spec(
 			continue;
 
 		/* In autotag mode, don't overwrite any locally-existing tags */
-		error = git_reference_create(&ref, remote->repo, refname.ptr, &head->oid, !autotag, 
+		error = git_reference_create(&ref, remote->repo, refname.ptr, &head->oid, !autotag,
```

Comment:
Given that this is a few hundred lines away from the actual change, I'd avoid unneccessary churn here.